functions.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import zipfile, os
+import glob
+client = boto3.client('cloudformation')
+s3 = boto3.resource('s3')
+
+
+DIRECTORY_IN_STR = "/numbers"
+
+class Functions(object):
+    #contains all the functions used in the project.
+  def __init__(self,parameter):
```

Comment:
Follow PEP8 coding principles. PyCharm IDE will guide you to do so.
```suggestion
  def __init__(self, parameter):
```

Reply:
done!